plutus-tx-plugin/src/PlutusTx/Options.hs

```diff
@@ -309,6 +310,9 @@ pluginOptions =
             rest <- many (alphaNumChar <|> char '_' <|> char '\\')
             pure (firstC : rest)
        in (k, PluginOption typeRep (plcParserOption p k) posCertify desc [])
+    , let k = "generate-callstack"
+          desc = "Generate callstack string and replace 'callStack' with callstack at the location."
```

Comment:
What does the "replace 'callStack' with callstack at the location." part mean for a compiler user?
I don't understand it myself, don't think that end-users will get it either.